symfony/assets/scripts/entry/general.ts

```diff
@@ -32,3 +33,43 @@ jQuery(() => {
     config.save();
   });
 });
+
+jQuery(() => {
+  const newsNavlink = jQuery("#navlink-news");
+
+  // If we're ON the events page, then never show a badge
+  if (newsNavlink.hasClass("active")) {
+    return;
+  }
+
+  // Validate that we have a timestamp for when events were last updated
+  const timestampStr = newsNavlink.data("latest-event");
```

Comment:
I understand that Static.ts/DATA_BRIDGE stuff cannot be used here, but I wanted to mention it just to make sure you are aware of it.

Reply:
Thank you! I did see that, and I thought at first about doing a similar approach. I went with the `data` attribute approach _mostly_ because it'd been a while since I'd used that technology (and it sounded fun), and then a little bit because it'd keep the data a bit more isolated. It won't be globally available/get in the way, the data is scoped just to this use case.

It was mostly the former reason tho haha